asm-patch/Kptfile

```diff
@@ -124,4 +124,13 @@ openAPI:
           - marker: us-central1-c
             ref: '#/definitions/io.k8s.cli.setters.gcloud.compute.location'
           - marker: PROJECT_ID
-            ref: '#/definitions/io.k8s.cli.setters.gcloud.core.project'
\ No newline at end of file
+            ref: '#/definitions/io.k8s.cli.setters.gcloud.core.project'
+    io.k8s.cli.substitutions.citadel-patch-base-dir:
+      type: string
+      x-k8s-cli:
+        substitution:
+          name: citadel-patch-base-dir
+          pattern: "../base-dir"
+          values:
+          - marker: base-dir
```

Comment:
```
          pattern: "../base"
          values:
          - marker: base
```

In the `pattern`, it is the actual value we put in the template.

Reply:
not needed.